scripts/hid_configurator/README.rst

```diff
@@ -82,7 +82,15 @@ Complete the following steps:
        In versions earlier than 5.44, the HID device attached by BlueZ could obtain wrong VID and PID values (ignoring values in Device Information Service), which would stop HIDAPI from opening the device.
        In versions earlier than 5.56, the HID device attached by BlueZ might provide incomplete HID feature report on get operation.
 
-   Additionally, to call the Python script on Linux without root rights, install the provided udev rule :file:`99-hid.rules` file by copying it to :file:`/etc/udev/rules.d` and replugging the device.
+#. In addition, if you don't want to use the root access to run the python script, copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d` and reconnect the device.
+#. If you want to connect to a device with a different Vendor or Product ID other than the one specified in the file, you can either run the script with the root permission or add a new entry to the :file:`99-hid.rules` file with your Vendor and Product ID, then copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d`  and reconnect the device.
```

Comment:
```suggestion
#. If you want to connect to a device with a different Vendor or Product ID other than the one specified in the file, you can either run the script with the root permission or complete the following step without root permission:
    
    a. Add a new entry to the :file:`99-hid.rules` file with your Vendor and Product ID.
    #. Copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d`.
    #. Reconnect the device.
    
    Vendor and Product ID can be specified in the configuration file related to the nRF Desktop application.
    The following example shows the entry to add to the :file:`99-hid.rules` file: 
    
    .. parsed-literal::
       :class: highlight

       ATTRS{idVendor}=="my Vendor ID", ATTRS{idProduct}=="my Product ID", MODE="0666", SYMLINK+="nrf52-desktop-my-app-name"
```
After offline discussion with greg, concluded that it a good idea to break down the long sentences into steps.